ds_judgements_public_ui/templates/pages/about_this_service.html

```diff
@@ -161,7 +161,7 @@ <h2 id="section-data">Judgments and decisions as data</h2>
   <p>
     Judgments and decisions are sent to us as Word documents using the National Archive’s Transfer Digital Records
     service.
-    This is the digital record of the publication version of the judgment. We then create a digital surrogates of the
+    This is the digital record of the publication version of the document. We then create a digital surrogates of the
```

Comment:
I renamed a few references of judgment to document in the static pages. There probably is more changes to these static pages that we will like to make detailing more specifics about press summaries but I think these generalisations are fine for now. Happy to revert if people want.